cst438_gradebook/src/main/java/com/cst438/services/RegistrationService.java

```diff
@@ -1,11 +1,28 @@
 package com.cst438.services;
 
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.web.client.RestTemplate;
+
 import com.cst438.domain.CourseDTOG;
 
+
 public class RegistrationService {
 	
+	@Value("${registration.url}")
+	private String registrationUrl;
+	
 	public void sendFinalGrades(int course_id , CourseDTOG courseDTO) {
-		
+		RestTemplate restTemplate = new RestTemplate();
```

Comment:
Looks like this part should belong to `RegistrationServiceREST`, wdyt?

Reply:
Making some changes and pushing a new commit after going the walkthrough video session